### ISSUE TITLE
[Bug]: List of files uploaded to File Drop contains unlabeled icons

### ISSUE THREAD

**Issue by rosahaj** [nextcloud/server#49923]
### ⚠️ This issue respects the following points: ⚠️

- [x] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [x] This issue is **not** already reported on [Github](https://github.com/nextcloud/server/issues?q=is%3Aopen+is%3Aissue+label%3Abug) OR [Nextcloud Community Forum](https://help.nextcloud.com/) _(I've searched it)_.
- [x] Nextcloud Server **is** up to date. See [Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) for supported versions.
- [x] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Bug description

After uploading files to a File Drop, an icon indicating the file type is displayed next to the file name (e.g. a small PDF icon for PDF files). Because the icon is neither labeled nor hidden from accessibility APIs, it's described as an unlabeled image when using certain screen readers together with certain browsers (such as Google TalkBack + Google Chrome).

**Relevant HTML snippet (after file upload):**
```html
<ul id="drop-uploaded-files">
  <li title="Example.pdf" data-name="Example.pdf">
    <img src="/apps/theming/img/core/filetypes/application-pdf.svg?v=1b3d5f7a"> Example.pdf
  </li>
</ul>
```

Since the file type is inferred based on the file extension alone and does not provide any new information not already contained in the filename, the icon is merely a visual aid. I believe the icon should therefore be hidden from accessibility APIs with `aria-hidden="true"`.

### Steps to reproduce

1. Create new file request
2. Upload a file to File Drop

### Expected behavior

Image is ignored by accessibility APIs

### Nextcloud Server version

30

### Operating system

None

### PHP engine version

None

### Web server

None

### Database engine version

None

### Is this bug present after an update or on a fresh install?

None

### Are you using the Nextcloud Server Encryption module?

None

### What user-backends are you using?

- [ ] Default user-backend _(database)_
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

### Configuration report

```json

```

### List of activated Apps

```shell

```

### Nextcloud Signing status

```shell

```

### Nextcloud Logs

```json

```

**Comment by rosahaj** [comment]
I noticed that the icon displayed next to the uploaded files not only indicates the file type, but can also indicate that the upload of a specific file has failed. Rather than hiding the icon from the accessibility tree, it could be labeled in accordance with the file type (for successfully uploaded files) or with an "upload failed" label (for unsuccessfully uploaded files).

**Comment by susnux** [comment]
Fixed in 30.0.6
